ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -528,6 +541,7 @@ static KsqlRestApplication buildApplication(
             new KsqlSecurityContextBinder(config, securityExtension, schemaRegistryClientFactory));
   }
 
+  @SuppressWarnings("checkstyle:MethodLength")
```

Comment:
file a follow up for fixing this? I understand doing that in this PR will balloon scope

Reply:
Actually, not needed. I had already fixed it but forgot the supress